packages/cmf/scripts/cmf-settings.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const pathLib = require('path');
+const mkdirp = require('mkdirp'); // eslint-disable-line import/no-extraneous-dependencies
+const deepmerge = require('deepmerge'); // eslint-disable-line import/no-extraneous-dependencies
+const program = require('commander'); // eslint-disable-line import/no-extraneous-dependencies
+const utils = require('./utils');
+
+program
+	.version('0.0.1')
+	.option('-d, --dev', 'dev sources instead of sources')
+	.option('-q, --quiet', 'display nothing')
+	// .option('-w, --watch', 'watch files')
+	.parse(process.argv);
+
+// 1 - Init some stuff to use next
+const findJsonInFolder = utils.findJsonInFolder;
+const concatMerge = utils.concatMerge;
+const overrideRoutes = utils.overrideRoutes;
+const overrideActions = utils.overrideActions;
+const log = utils.getLogger(program.quiet);
+const configurations = [];
+const cmfconfig = utils.getCmfConfig();
+
+// 2 - Get sources & destination paths
+const sources = program.dev
+	? cmfconfig['cmf-settings']['dev-sources']
+	: cmfconfig['cmf-settings'].sources;
+const destination = `${process.cwd()}/${cmfconfig['cmf-settings'].destination}`;
```

Comment:
use the librairies path to manage the / on windows, linux
https://nodejs.org/api/path.html